charts/nango/values.yaml

```diff
@@ -45,6 +45,7 @@ persist:
   name: persist
   url: http://nango-persist
   replicas: 1
+  tag: e6fd036d8bf2f5b7f9a8cd574746821e72b3801e # 2024/06/07
```

Comment:
I wonder what kind of release strategy we should have to update this

Reply:
I don't know either, I just think it's better to pin but yeah we need to do it regularly

Reply:
We could on new image update trigger an action in this repo which pushes a new tag version. I've used https://github.com/peter-evans/repository-dispatch with success in the past